src/main/java/com/github/igotyou/FactoryMod/utility/FactoryModGUI.java

```diff
@@ -168,6 +168,21 @@ private IClickable getSetupClick(FurnCraftChestEgg factory) {
 		return new LClickable(is, p -> showFactoryCreation(factory, true));
 	}
 
+	private IClickable getFuelClick(FurnCraftChestEgg factory, InputRecipe recipe) {
+		if (recipe == null) {
+			return null;
+		}
+		if (factory == null) {
+			return null;
+		}
+		ItemStack is = factory.getFuel();
+		ItemAPI.setDisplayName(is, ChatColor.GOLD + "Fuel cost for recipe");
+		List<String> lore = new ArrayList<>();
+		lore.add(ChatColor.AQUA + "- " + recipe.getTotalFuelConsumed() + " " + ItemNames.getItemName(is.getType()));
+		ItemAPI.setLore(is, lore);
+		return new LClickable(is, p -> showRecipeFor(factory, recipe, false));
```

Comment:
Why should clicking the fuel show the recipe you are already viewing? Just make it a DecorationStack and not do anything at all